src/styles/header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+}
```

Comment:
not sure bout this one, you can prolly move it to separate file too

Reply:
or use logo as an element, not as a separate block